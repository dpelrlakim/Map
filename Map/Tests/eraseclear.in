newInit m
erase m 1
erase m 7
erase m 2
size m
[]= m 2 -100
erase m 2
clear m
empty m
