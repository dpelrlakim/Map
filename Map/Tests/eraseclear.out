m: (1,2) (3,4) (5,6) (7,8)
m: (3,4) (5,6) (7,8)
m: (3,4) (5,6)
m: (3,4) (5,6)
2
m: (3,4) (5,6)
m: (2,-100) (3,4) (5,6)
m: (3,4) (5,6)
m:
1
m:
